inspire/dojson/common/base.py

```diff
@@ -39,6 +39,12 @@ def control_number(self, key, value):
     return value[0]
 
 
+@hep2marc.over('001', 'control_number')
+def control_number2marc(self, key, value):
+    """Record Identifier."""
+    return [value]
```

Comment:
`controlfield` are per definition unique. So this shall really be reflected also in `create_record()`